src/System.IO.Compression/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
Does this mean we will build and run all the tests twice (with the exception of the newly added tests, which will only run once on netcoreapp)?